index.bs

```diff
@@ -470,6 +474,8 @@ Every {{Document}} has a <dfn for="Document">post-prerendering activation steps
 
       1. Let |doc| be |bc|'s [=active document=].
 
+      1. Set |doc|'s [=Document/activation start time=] to |activation start time|.
```

Comment:
Added the same origin check, same as [`update the session history with the new page`](https://html.spec.whatwg.org/multipage/browsing-the-web.html#update-the-session-history-with-the-new-page) logic does [here](https://html.spec.whatwg.org/multipage/browsing-the-web.html#:~:text=newdocument's%20origin%20is%20the%20same%20as%20previousdocument's%20origin) for unload timing.


> Also it means documents that didn't exist before prerendering don't have an activation start time, which might be a bit inconsistent/racy. It might be OK to just provide activation start time for the top level document and subframes don't have it to start with?

I don't understand what is problem here.
If my understanding is correct, `prerenderingchange` events is called if and only if the document has been created while prerendering. What is problem if we set the activation start time if and only if the document has been created while prerendering?

Reply:
Yeah I think this can be OK probably. I had just imagined it could appear racy: suppose a prerendered page makes many subframes dynamically. Whether they have an activation start time would depend on whether activation started quickly enough, which seemed inconsistent. However I see now the timestamp is relative to the  time origin of the document, so it'd be negative if the document wasn't created yet, which is strange also. So I can be OK with the current patch.